### PR TITLE
(BOLT-31) PDK update

### PR DESCRIPTION
**Pull request by LukasAud**
Running a PDK update command to ensure metadata and Gemfile dependencies keep in line with current templates.

Tested locally. Tests passing.

### CODE REVIEW

File: .fixtures.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# This file can be used to install module dependencies for unit testing
+# See https://github.com/puppetlabs/puppetlabs_spec_helper#using-fixtures for details
+---
+fixtures:
+  forge_modules:
+#     stdlib: "puppetlabs/stdlib"
```
Comment (pmcmaw) [inline .fixtures.yml]: If the stdlib dependency is commented out, it may be as well just taking it out.

**Review by pmcmaw — COMMENTED** [review]
One minor suggested adjustment.

File: .fixtures.yml (hunk 1 above)
Comment (LukasAud) [reply .fixtures.yml]: addressed

**Review by gavindidrichsen — APPROVED** [review]
Nice work.

**Pull request merged**